CentralComputing/PRUManager.cpp

```diff
@@ -126,4 +126,17 @@ void PRUManager::initialize_sensor_error_configs() {
 }
 
 void PRUManager::check_for_sensor_error(const std::shared_ptr<PRUData> & check_data) {
+  // hardcoded for two of each type of sensor right now, could use standard dev?
+  // just using distance for now, as velocity is based off of distance
```

Comment:
To be pedantic about how the PRU calculates everything....

I'll call it a "Pulse" when there is a passing tape.

It keeps track of 3 metrics:
1. Time between last two Pulses.  (e.g. 1 second between the last to pulses)
2. Time since last Pulse (e.g. 0.5 seconds since a pulse). 
  * *This is very useful when the Pod is moving slowly.* Say you roll forward quickly just a bit, such that you have 0.1 second between the last two pulses. But then you stop. It will display your speed as a constant (distance/0.1 == a decently high velocity), but as soon as the Time Since Last Pulse goes over 0.1 seconds, you know that your speed is now decreasing, and can calculate it appropriately.
3. Number of Pulses. 

Position is based on number of pulses (3)

Velocity is based on the timing of (1) and (2), and which one represents a _slower_ speed. (it is of course representative of the number of pulses, so if the detection of pulses is off, the velocity will be off )